packages/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift

```diff
@@ -2,7 +2,7 @@ import UIKit
 import Dispatch
 import Foundation
 
-var subscribers = [ExpoAppDelegateSubscriberProtocol]()
+var subscribers = [ExpoAppDelegateSubscriber]()
```

Comment:
You'll need to revert this. We do want to use `ExpoAppDelegateSubscriberProtocol` here. I guess you had compilation errors without that change, but you should rather add `func buildMenu(with builder: UIMenuBuilder)` to the protocol instead.

```suggestion
var subscribers = [ExpoAppDelegateSubscriberProtocol]()
```

Reply:
Gotcha 👍 But I'm a bit curious as to why it needs to be a protocol here?

Reply:
We just prefer protocol-oriented approach and since it's already done that way, it's better not to change if not needed 😉

Reply:
I'm not sure I can add it like that since the protocol in some places is used together with the UIResponder, I get type clashes on the buildMenu method. Maybe you could give it a try?